add exports content and rn (#4689)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes/72-Feb-2024/exports-rn.md

> ---

Reviewer: consider naming this as just `exports.md` . 

i was using `-rn.md` but recently got good feedback that it's a little redundant in the URL `/docs/dbt-versions/release-notes/.../-rn` so have stopped

title: "New: Use exports to materialize saved queries"

> description: "February 2024: Use exports to materialize and schedule saved queries with dbt Cloud, and integrate with additional tools."

Reviewer: ```suggestion
description: "February 2024: Use the exports feature to materialize and schedule saved queries with dbt Cloud, and to integrate with additional tools."
```

sidebar_label: "New: Use Exports to materialize Saved Queries"
sidebar_position: 09
tags: [Feb-2024]

> date: 2024-02-08

Author: putting this date but may change

> date: 2024-02-08

Author: ```suggestion
date: 2024-02-07
```

---

> You can now use [exports](/docs/use-dbt-semantic-layer/exports) with the [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl), allowing you to query reliable metrics and fast data reporting. Exports enhance the saved queries feature, allowing you to materialize commonly used queries directly within your data platform using dbt Cloud's Job scheduler.

Reviewer: ```suggestion
You can now use the [exports](/docs/use-dbt-semantic-layer/exports) feature with [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl), allowing you to query reliable metrics and fast data reporting. Exports enhance the saved queries feature, allowing you to materialize commonly used queries directly within your data platform using dbt Cloud's job scheduler.
```

> By exposing tables of metrics and dimensions, exports provide an integration path by enabling you to integrate with additional tools that don't natively connect with the dbt Semantic Layer, such as PowerBI.

Reviewer: ```suggestion
By exposing tables of metrics and dimensions, exports enable you to integrate with additional tools that don't natively connect with the dbt Semantic Layer, such as PowerBI.
```

> Exports are available on dbt Cloud [Team or Enterprise](https://www.getdbt.com/pricing/) plans, on dbt versions 1.7 or higher.

Reviewer: ```suggestion
Exports are available for dbt Cloud [Team or Enterprise](https://www.getdbt.com/pricing/) plans on dbt versions 1.7 or newer.
```
